app/src/main/res/values/strings.xml

```diff
@@ -79,4 +79,15 @@
     <string name="lat_std_dev_threshold" translatable="false">Latitude Standard Dev Threshold</string>
     <string name="add_note_to_tree">Add note to tree</string>
     <string name="note">NOTE</string>
+
+    <!--Strings for the permission management and rationale-->
+    <string name="accept_camera_permission_header">Accept Camera permission</string>
+    <string name="accept_camera_permission_message">Camera is needed for this app to allow it capture pictures, accept this permission to use this device to take pictures</string>
```

Comment:
nit; I would maybe try to make this text more concise, something like "Camera is needed for Treetracker to capture pictures, please accept this permission"